src/js/definitions/currency.js

```diff
@@ -1,20 +1,24 @@
 import CurrencyWidget from '../widgets/CurrencyWidget';
 import CurrencyReviewWidget from '../review/CurrencyWidget';
+import { validateCurrency } from '../validation';
 
 function schema() {
   return {
     type: 'number'
   };
 }
 
-function uiSchema(title) {
+function uiSchema(title = 'Dollar amount') {
```

Comment:
I added a default title because that's what most other definitions have. We should probably make them all consistent at some point, but figured I would add this while working in this file. It won't break any existing uses of the currency definition because they all include a title now, which will overwrite the default one added here.